tap-lightcast/config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "client_id": "",
+    "client_secret": "",
+    "limit": -1 /* set to -1 only if no IDs limit is required */
```

Comment:
I didnt include a limit in my config and the extractor failed:
`2023-05-03T21:59:28.843824Z [info     ] singer_sdk.exceptions.ConfigValidationError: Config validation failed: 'limit' is a required property`